unix-in.lisp

```diff
@@ -19,6 +19,8 @@
 (named-readtables:in-readtable :standard)
 
 (defvar *post-command-hook* (make-instance 'nhooks:hook-void))
+(defvar *path-warning* 1
```

Comment:
Can you use t instead of 1 for the default value?

Reply:
I'd started with `t` but got weird errors on the REPL -- I think because there was a `t` executable on my path